march_hardware/src/PDOmap.cpp

```diff
@@ -8,124 +8,143 @@ namespace march4cpp
 {
 PDOmap::PDOmap()
 {
-  this->initAllObjects();
+  this->allObjects[IMCObjectName::StatusWord] = IMCObject(0x6041, 16);
+  this->allObjects[IMCObjectName::ActualPosition] = IMCObject(0x6064, 32);
+  this->allObjects[IMCObjectName::MotionErrorRegister] = IMCObject(0x2000, 16);
+  this->allObjects[IMCObjectName::DetailedErrorRegister] = IMCObject(0x2002, 16);
+  this->allObjects[IMCObjectName::DCLinkVoltage] = IMCObject(0x2055, 16);
+  this->allObjects[IMCObjectName::DriveTemperature] = IMCObject(0x2058, 16);
+  this->allObjects[IMCObjectName::ActualTorque] = IMCObject(0x6077, 16);
+  this->allObjects[IMCObjectName::CurrentLimit] = IMCObject(0x207F, 16);
+  this->allObjects[IMCObjectName::MotorPosition] = IMCObject(0x2088, 32);
+  this->allObjects[IMCObjectName::ControlWord] = IMCObject(0x6040, 16);
+  this->allObjects[IMCObjectName::TargetPosition] = IMCObject(0x607A, 32);
+  this->allObjects[IMCObjectName::TargetTorque] = IMCObject(0x6071, 16);
+  this->allObjects[IMCObjectName::QuickStopDeceleration] = IMCObject(0x6085, 32);
+  this->allObjects[IMCObjectName::QuickStopOption] = IMCObject(0x605A, 16);
 }
 
-void PDOmap::addObject(IMCObjectName objectname)
+void PDOmap::addObject(IMCObjectName objectName)
 {
-  if (this->allObjects.count(objectname) != 1)
+  if (this->allObjects.count(objectName) != 1)
   {
     ROS_WARN("IMC object does not exist (yet), or multiple exist");
     return;
   }
-  else if (this->PDOObjects.count(objectname) != 0)
+  else if (this->PDOObjects.count(objectName) != 0)
   {
     ROS_WARN("IMC object is already added to PDO map");
     return;
   }
   else
   {
-    this->PDOObjects[objectname] = this->allObjects[objectname];
+    this->PDOObjects[objectName] = this->allObjects[objectName];
   }
 }
 
 std::map<enum IMCObjectName, int> PDOmap::map(int slaveIndex, enum dataDirection direction)
 {
   this->sortPDOObjects();
-  int reg;
-  int SMAddress;
+
   if (direction == dataDirection::miso)
   {
-    reg = 0x1A00;
-    SMAddress = 0x1C13;
+    ROS_INFO("Mapping miso");
```

Comment:
This info is a bit useless as it gives no detailed information, and if it does it should be `ROS_DEBUG`

Reply:
removed, redundant statements